backend/api/user.go

```diff
@@ -50,25 +75,65 @@ func Login(c *gin.Context) {
 	user, err = model.UserInfoByName(username)
 
 	if err != nil {
-		c.JSON(http.StatusOK, gin.H{"result": "fail"})
+		c.JSON(http.StatusNotFound, gin.H{"result": "no user found"})
 		return
 	}
 
 	if err := pkg.Compare(user.Password, password); err != nil {
-		c.JSON(http.StatusOK, gin.H{"result": "wrong password"})
+		c.JSON(http.StatusNotFound, gin.H{"result": "wrong password"})
```

Comment:
Should not provide info for use existence.

Reply:
I don't get it QQ

Reply:
I think letting user know wheter a username exists or not may cause a security issue. But since your username is unique and this information can be retrieved by registering a new user, you can leave this.

Reply:
If you leave the "wrong password" message, then 401 is better.

Reply:
Oh I see